Sources/PointFree/Account/Account.swift

```diff
@@ -53,6 +53,7 @@ private func fetchAccountData<I>(
     .flatMap(Current.stripe.fetchSubscription)
 
   let upcomingInvoice = stripeSubscription
+    .flatMap { $0.isRenewing ? pure($0) : throwE(unit) }
```

Comment:
Noticed that we were 404ing on the dashboard whenever a subscriber with a cancelled/cancelling account hit their account page. This should reduce some noise.